src/sqlancer/mariadb/gen/MariaDBSetGenerator.java

```diff
@@ -35,11 +35,11 @@ private enum Action {
 
         AUTOCOMMIT("autocommit", (r) -> 1, Scope.GLOBAL, Scope.SESSION), //
         BIG_TABLES("big_tables", (r) -> Randomly.fromOptions("OFF", "ON"), Scope.GLOBAL, Scope.SESSION), //
-        COMPLETION_TYPE("completion_type", (r) -> Randomly.fromOptions("'NO_CHAIN'", "'CHAIN'", "'RELEASE'", 0, 1, 2),
+        COMPLETION_TYPE("completion_type", (r) -> Randomly.fromOptions("'NO_CHAIN'", "'CHAIN'", "'RELEASE'", "0", "1", "2"),
```

Comment:
What's the motivation for these changes? Any IDE warning or so?

Reply:
hi! yes, warnings in intelliJ. I will resume with the PR soon, another priority kicked in :), you know how these things are.